timeseries-spi-impl/sos-series-dao/src/main/java/org/n52/series/api/v1/db/da/TimeseriesRepository.java

```diff
@@ -106,7 +106,13 @@ public TimeseriesRepository(ServiceInfo serviceInfo) {
             SeriesDao seriesDao = new SeriesDao(session);
             List<TimeseriesMetadataOutput> results = new ArrayList<TimeseriesMetadataOutput>();
             for (SeriesEntity timeseries : seriesDao.getAllInstances(query)) {
-                results.add(createCondensed(timeseries, query));
+            	/*
+            	 *  ATM, the SWC REST API only supports numeric types
+            	 *  We check for a unit to check for them
+            	 */
+            	if (timeseries.getUnit() != null) {
+            		results.add(createCondensed(timeseries, query));
+            	} 
```

Comment:
:+1: but on which level should be log, DEBUG or INFO?

In addition, we might add a log statement before line 158new, too. Or we add a second if statement with another exception message.